modules/navier_stokes/test/tests/finite_volume/ins/channel-flow/segregated/2d/2d-segregated-energy.i

```diff
@@ -0,0 +1,257 @@
+mu = 2.6
+rho = 1.0
+k = 5.0
+cp = 700
+alpha = 150
+advected_interp_method = 'average'
+velocity_interp_method = 'rc'
+
+pressure_tag = "pressure_grad"
+
+[Mesh]
+  [mesh]
+    type = CartesianMeshGenerator
+    dim = 2
+    dx = '0.3'
+    dy = '0.3'
+    ix = '3'
+    iy = '3'
+  []
+[]
+
+[GlobalParams]
+  rhie_chow_user_object = 'rc'
+[]
+
+[Problem]
+  nl_sys_names = 'u_system v_system pressure_system energy_system'
+  previous_nl_solution_required = true
+  error_on_jacobian_nonzero_reallocation = true
+[]
+
+[UserObjects]
+  [rc]
+    type = INSFVRhieChowInterpolatorSegregated
+    u = vel_x
+    v = vel_y
+    pressure = pressure
+  []
+[]
+
+[Variables]
+  [vel_x]
+    type = INSFVVelocityVariable
+    initial_condition = 0.5
+    nl_sys = u_system
+    two_term_boundary_expansion = false
+  []
+  [vel_y]
+    type = INSFVVelocityVariable
+    initial_condition = 0.0
+    nl_sys = v_system
+    two_term_boundary_expansion = false
+  []
+  [pressure]
+    type = INSFVPressureVariable
+    nl_sys = pressure_system
+    initial_condition = 0.2
+    two_term_boundary_expansion = false
+  []
+  [T_fluid]
+    type = INSFVEnergyVariable
+    initial_condition = 300
+    nl_sys = energy_system
+    two_term_boundary_expansion = false
```

Comment:
I saw @GiudGiud comment on this. Curious why we dislike the `two_term_boundary_expansion`

Reply:
We don't dislike it, it just allows direct comparison with other cfd tools which operate with these assumptions.